core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -120,10 +120,15 @@ public class RestConfig extends AbstractConfig {
 
   public static final String ACCESS_CONTROL_ALLOW_HEADERS = "access.control.allow.headers";
   protected static final String ACCESS_CONTROL_ALLOW_HEADERS_DOC =
-      "Set value to Jetty Access-Control-Allow-Origin header for specified headers. "
+      "Set value to Jetty Access-Control-Allow-Headers header for specified headers. "
       + "Leave blank to use Jetty's default.";
   protected static final String ACCESS_CONTROL_ALLOW_HEADERS_DEFAULT = "";
 
+  public static final String ACCESS_CONTROL_EXPOSE_HEADERS = "access.control.expose.headers";
+  protected static final String ACCESS_CONTROL_EXPOSE_HEADERS_DOC =
+      "set value to Jetty Access-Control-Expose-Headers header for specified headers.";
```

Comment:
Can we add a comment like above it it applies here? `"Leave blank to use Jetty's default."`

Lgtm otherwise.

Reply:
Thank you! Just added it, may I get a review again?